scripts/build.js

```diff
@@ -66,7 +66,6 @@ files.forEach((filePath) => {
     const dataAsHTMLElement = getElement({ selector: 'div', htmlString: `<div>${rawData}</div>` })
     const { name } = getNameAndSize(filePath)
     const title = descriptions[pascalCase(name)]
-    if (title) dataAsHTMLElement.querySelector('svg').prepend(`<title>${title}</title>`)
```

Comment:
Need to remove it here because we are adding it when building the react. (Will be handled for vue and elements in separate PRs)

Reply:
With this fix the icons that are built in dist, the optimized ones, will loose title. I've seen that those icons were used in FINN directly. SO those will loose title. Can it be good to at least keep the english one? I know that those should not be used directly but in case they are, maybe we can keep english titles to be on the safe side?

I'll try to find examples of those usages

Reply:
https://github.schibsted.io/finn/new-construction-frontend/blob/457066bdf756d2aaa189382741661649b8b95a88/packages/ad/src/components/keyInfo/index.jsx this looks like one case of those usages

Reply:
Hmmm good point. We can wait to merge this until we also handle Vue and Elements... Or wdyt?
Because if I remove this change, then the React icons will have two titles.